### PR TITLE
Docs: update DG UML diagrams

### PR DESCRIPTION
**Pull request by gigituang**

### CODE REVIEW

File: docs/umlDiagrams/ArchitectureDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Main -[#lightgrey]-> UI
+Main -[#lightgrey]-> Logic
+Main -[#lightgrey]-> Storage
+Main -up[#lightgrey]-> Model
+Main -down[hidden]-> Commons
+
+Storage -up[STORAGE_COLOR].> Model
+Storage .right[STORAGE_COLOR].>File
+User ..> UI
+@enduml
```
Comment (yusufaine) [inline docs/umlDiagrams/ArchitectureDiagram.puml:34]:
```suggestion
@enduml

```

File: docs/umlDiagrams/ArchitectureSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+storage --[STORAGE_COLOR]> logic
+deactivate storage
+
+logic --[LOGIC_COLOR]> ui
+deactivate logic
+
+ui--[UI_COLOR]> user
+deactivate ui
+@enduml
```
Comment (yusufaine) [inline docs/umlDiagrams/ArchitectureSequenceDiagram.puml:39]:
```suggestion
@enduml

```

File: docs/umlDiagrams/ClassDiagrams.puml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+RecipeBook<|-- Recipe
+Recipe <|-- Name
+Recipe <|-- Ingredient
+Recipe <|-- ServingSize
+Recipe <|-- Steps
+Recipe <|-- CompletionTime
+Iterable <|.. UniqueRecipeList
+
+@enduml
```
Comment (yusufaine) [inline docs/umlDiagrams/ClassDiagrams.puml:80]:
```suggestion
@enduml

```

**Review by yusufaine — CHANGES_REQUESTED** [review]
EOF line update

**Review by yusufaine — APPROVED** [review]
LGTM, good job on the UML diagram and styling 👍

**Review by aidanaden — APPROVED** [review]
LGTM! 🍝 🍝 🍝

**Pull request merged**